opencog/benchmark/benchmark.py

```diff
@@ -4,10 +4,15 @@
 """
 Python binding benchmarks
 
-Displays the number of operations per second achieved under various scenarios
-utilizing the Python bindings that wrap the AtomSpace using Cython. Some of the
-tests compare the evaluation of nodes using the Python Scheme bindings which
-take much longer than their comparable Cython bindings.
+Displays the number of operations per second achieved under various
+scenarios utilizing the Python bindings that wrap the AtomSpace using
+Cython. Some of the tests compare the evaluation of nodes using the
+Python Scheme bindings which take much longer than their comparable
+Cython bindings. XXX Which is utterly false and insane -- the
+measurements are being done incorrectly, since the Scheme bindings
+normally run about 3x to 10x faster than the Python bindings do.
+Python is generally quite slow, and is very unlikely to be faster than
+scheme; something is borked with this test case.
```

Comment:
I think this was poorly worded and was referring to the scheme eval overhead not the scheme bindings.